Formula/d/dbml-cli.rb

```diff
@@ -14,6 +14,16 @@ class DbmlCli < Formula
   def install
     system "npm", "install", *std_npm_args
     bin.install_symlink Dir["#{libexec}/bin/*"]
+
+    # Remove incompatible pre-built binaries
+    os = OS.kernel_name.downcase
+    arch = Hardware::CPU.intel? ? "x64" : Hardware::CPU.arch.to_s
+    node_modules = libexec/"lib/node_modules/@dbml/cli/node_modules"
+    node_modules.glob("oracledb/build/Release/oracledb-*.node").each do |f|
+      rm(f) unless f.basename.to_s.match?("#{os}-#{arch}")
+    end
+
+    deuniversalize_machos node_modules/"fsevents/fsevents.node" if OS.mac?
```

Comment:
let's also link these `deuniversalize_machos` to 

- https://github.com/Homebrew/brew/pull/21136

moving forward